app/views/jobseekers/job_applications/build/references.html.slim

```diff
@@ -7,19 +7,19 @@
     .govuk-caption-m = t("jobseekers.job_applications.current_step", current: process_steps.current_step_number, total: 10)
     h1.govuk-heading-l = t(".heading")
 
-    p.govuk-body = t(".references_required_html")
-    p.govuk-body = t(".no_children")
+    p.govuk-body = t(".description1_html")
+    p.govuk-body = t(".description2")
 
-    - if job_application.references.none?
-      = render Shared::NotificationComponent.new(content: { title: t(".no_references"), body: govuk_link_to(t("buttons.add_reference"), new_jobseekers_job_application_build_detail_path(job_application, :references), button: true, class: "govuk-!-margin-bottom-0") }, style: "empty", dismiss: false, background: true, alert: false)
+    - if job_application.references.any?
+      = render Jobseekers::JobApplications::DetailComponent.with_collection(job_application.references, title_attribute: "name", info_to_display: reference_info)
+      = govuk_link_to t("buttons.add_another_reference"), new_jobseekers_job_application_build_detail_path(job_application, :references), button: true, class: "govuk-button--secondary"
     - else
-      = render Jobseekers::JobApplications::DetailComponent.with_collection(job_application.references, info_to_display: reference_info)
-      = govuk_link_to t("buttons.add_reference"), new_jobseekers_job_application_build_detail_path(job_application, :references), button: true, class: (job_application.references.count > 1 ? "govuk-button--secondary" : "")
+      = render Shared::NotificationComponent.new(content: { title: t(".no_references"), body: govuk_link_to(t("buttons.add_reference"), new_jobseekers_job_application_build_detail_path(job_application, :references), button: true, class: "govuk-!-margin-bottom-0") }, style: "empty", dismiss: false, background: true, alert: false)
 
     = form_for @form, url: wizard_path, method: :patch do |f|
       = f.govuk_error_summary
 
-      - if job_application.references.count > 1
+      - if job_application.references.size > 1
```

Comment:
how do you feel about `job_application.references.many?`

Reply:
Does pretty much the same thing under the hood and reads nicer so it's a ✅  from me